packages/workflow/src/Expression.ts

```diff
@@ -44,14 +48,27 @@ export class Expression {
 		this.workflow = workflow;
 	}
 
+	static resolveWithoutWorkflow(expression: string) {
+		return tmpl.tmpl(expression, {});
+	}
+
 	/**
 	 * Converts an object to a string in a way to make it clear that
 	 * the value comes from an object
 	 *
 	 */
 	convertObjectValueToString(value: object): string {
 		const typeName = Array.isArray(value) ? 'Array' : 'Object';
-		return `[${typeName}: ${JSON.stringify(value)}]`;
+
+		if (DateTime.isDateTime(value)) {
```

Comment:
Sorry for the confusion but please check if this can pass if `value` is invalid `DateTime` object. We still may need `invalidReason` check here.

Reply:
My bad, I misunderstood!